so that the network tests are only ran if networking is up
(debian/rules has the other half)

--- a/t/90_regression/agaran-filter-httpd.t
+++ b/t/90_regression/agaran-filter-httpd.t
@@ -5,6 +5,10 @@
 use strict;
 
 BEGIN {
+  unless (-f 'run_network_tests') {
+	  print "1..0 # skip - Network access (and permission) required to run this test\n";
+	  exit;
+  }
   eval "use HTTP::Request";
   if ($@) {
     print "1..0 # skip - HTTP::Request needed to test POE::Filter::HTTPD\n";
